src/test/extension.test.ts

```diff
@@ -12,4 +12,17 @@ suite('Extension Test Suite', () => {
 		assert.strictEqual(-1, [1, 2, 3].indexOf(5));
 		assert.strictEqual(-1, [1, 2, 3].indexOf(0));
 	});
+
+	test('Key Value store', () => {
+		const kvStore = new kvstore.KVStore('aderyn-vs-code-tests.json');
+		kvStore.set('/absolute/path/to/project1', {
+			scope: "src/",
+			exclude: "lib/",
+		});
+		assert.strictEqual(kvStore.get('/absolute/path/to/project1').scope, 'src/');
+
+		kvStore.set('/absolute/path/to/project2', {});
+		assert.strictEqual(kvStore.get('/absolute/path/to/project2').scope, undefined);
+	});
+
```

Comment:
This test passes locally for me. I am not sure CI can work this out because it opens a new vscode window to run this and then auto closes.